### DISCUSSION TITLE
OpenTTD Summit Idea (seriously)

### DISCUSSION THREAD

**Discussion by perezdidac** [OpenTTD/OpenTTD#9010] [General]
Hi all developers, artists and everyone,

The idea of organizing an OpenTTD summit in Hawaii sounds pretty much impossible, no matter how cool it would be.

But I would totally want to organize a remote one. And yes I volunteer to push for it and get participants for the ideas below. I wanted to hear your thoughts about this. Here are some ideas I have, please add more.

- Day 1 - Welcome to OpenTTD
    - History of the game, since TT until now
    - Game basics
    - Advance game features
    - History of NewGRF
    - Patches
- Day 2 - NewGRF
    - Bananas
    - NML workshop
    - Drawing workshop
    - AI/script creation workshop
- Day 3 - Development
    - Building OpenTTD
    - Workshops around making changes on different aspects of the game
- Day 4 - Unconference

It could be all recorded since there are too many time zones out there. We can also do screenshot/gifs competition, even play OpenTTD!

Now, I know it sounds crazy, so please either roast me for throwing this idea out or be helpful instead!

**Comment by frosch123** [comment]
The "developer live streams" have a similar goal.

However
* detailed talks require lots of preparation,
* even though the 1st and 2nd live stream only touched some things at the surface, they still took hours to prepare. (the 3rd one didn't, by concept :) )
* for many of the topics you mention, there are not even any good blogs/texts/youtube videos or similar. To take one example: the NML tutorial was a one person effort several years ago, it has not been updated since.

Anyway, if you find some people with knowledge to share, we happily invite them to the developer live streams: we are looking for more topics and faces. The "developer" in "developer live stream" refers to "who hosts them", not "who is on camera" :)
